docker/containerbuild.sh

```diff
@@ -63,15 +63,21 @@ mv /var/tmp/"$KSNAME"-docker.tar.xz $BUILDROOT/docker/
 
 # Create a Dockerfile to go along with the rootfs.
 
+BUILDDATE_RFC3339="$(date -d $BUILDDATE --rfc-3339=date)"
```

Comment:
```suggestion
BUILDDATE_RFC3339="$(date -d $BUILDDATE --rfc-3339=seconds)"
```

Whoops, I think this should be `seconds` instead. I don't think `ns` is necessary, but we could use it?

```
# date --rfc-3339=ns
2019-10-04 14:46:53.501343000+02:00
# date --rfc-3339=seconds
2019-10-04 14:46:56+02:00
# date --rfc-3339=date
2019-10-04
```